doc/classes/SkeletonProfileHumanoid.xml

```diff
@@ -5,6 +5,63 @@
 	</brief_description>
 	<description>
 		A [SkeletonProfile] as a preset that is optimized for the human form. This exists for standardization, so all parameters are read-only.
+		A humanoid skeleton profile is structured as follows:
+		[codeblock]
+		Root
+		└─ Hips
```

Comment:
From Tokage during the animation meeting:
I think we could specify the joint rule(role)
For example, bend: +X, roll +Y

Reply:
I'm not sure what the "joint rule" is. How can I obtain it for each bone?

Reply:
See also https://github.com/TokageItLab/realtime_retarget?tab=readme-ov-file#joint-rules .

The rest transform (joint rule) of a bone is determined by the following two definitions:

1. Godot Humanoid points from parent to child +Y as long as the bones are in a 1-1 relationship. Note: The default setting in Blender enforces this. It is implicitly handled internally to point from Head to Tail +Y.
2. Godot Humanoid determines the Y rotation of the rest transform so that the +X rotation in the pose rotation is Bend. This Y rotation is the roll of the edit bone in Blender.

The above two definitions determine the rest transform of the bone.

At this point, the role of each joint's rotation is as follows:

Spines
|Bone| Bend to internal | Roll to right | Bow to right |
| :-- | :--: | :--: | :--: |
|Hips|+X|-Y|+Z|
|Spine|+X|-Y|+Z|
..
..

Arms
|Bone| Bend to internal | Roll to internal | Spread to internal |
| :-- | :--: | :--: | :--: |
|LeftUpperArm|+X|+Y|+Z|
|LeftLowerArm|+X|+Y|-Z|
..
..

The terms bend and roll are not consistent with anatomy for some bones since anatomy may not define them for some bones.

Reply:
Also you can write down the list also for the OpenXRHand. See OpenXRHand.